src/pages/SendSwapRequest/Select.tsx

```diff
@@ -114,7 +114,7 @@ function Select({
                 </Grid>
               );
             }
-            case ParameterKind.Address: {
+            default: {
```

Comment:
:question: Was this necessary?

Reply:
@dakami I don't remember doing this change. Any comments on that?

Reply:
@thomaseizinger as far as I remember there was a Typescript warning because no default was specified so I changed it. It is a little confusing though, because it is not necessarily the default scenario. If it makes it more readable we can change it back.

Reply:
Will have to look at it but my gut feeling is that it is now functionally wrong.

Reply:
adapted to specify the Address case and a separate default that renders a `TextField`.